contrib/docker/solr/schema.xml

```diff
@@ -1 +0,0 @@
-../../../ckan/config/solr/schema.xml
```

Comment:
why are you duplicating this file now? For the solr schema we need a way to keep this file up to date with the current ckan solr schema, but also allow users to customize their solr schema.

How about using a volume shared between the ckan docker image and the solr one?